internal/projection/valkey_commands.go

```
@@ -94,21 +99,35 @@ func (v *Store) SetCommand(ctx context.Context, dto data.CommandChangedDTO) erro
+			return cmds, nil
+		}
+		return nil, err
+	}
 	if old == "" {
-		return cmds
+		return cmds, nil
 	}
 	var prev CommandView
 	if codec.Unmarshal([]byte(old), &prev) != nil || len(prev.Aliases) == 0 {
-		return cmds
+		return cmds, nil
```

golangci-lint (2.13.2)

[error] error is not nil (line 123) but it returns nil

(nilerr)